tests/solvers/test_composite_solvers.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import numpy as np
+
+from placeholder.solvers.composite import MultipleInitializations
+from placeholder.solvers.base import ScipyOpt
+from models import Rosenbrock
+
+
+@pytest.fixture
+def rb():
+    return Rosenbrock()
+    
+
+def test_multi_init(rb):
+    num_init = 3
+    xs_init = np.random.uniform(
+        low=[b[0] for b in rb.bounds],
+        high=[b[1] for b in rb.bounds],
+        size=(num_init, rb.n_dim),
+    )
+    sample_fun = lambda x: xs_init
+    solver = MultipleInitializations(sample_fun)
+    solver.solvers = [ScipyOpt()]
```

Comment:
Nice! Was going to add we should put in tests for composite solvers but you already did!